crm_properties.go

```diff
@@ -0,0 +1,107 @@
+package hubspot
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	crmPropertiesPath = "properties"
+)
+
+type CrmPropertiesList struct {
+	Results []CrmProperty
+}
+
+type CrmProperty struct {
+	Calculated       bool   `json:"calculated"`
```

Comment:
As mentioned in the comments, if zero values are a concern, it would be a good idea to use a pointer.
https://github.com/belong-inc/go-hubspot/pull/17#discussion_r1143540387